src/app.config.json

```
@@ -191,6 +191,7 @@
         }
       ]
     },
+    "triggeredOnChange": false,
```

We need a custom name to distinguish between ADF and ACA settings I think.
```suggestion
    "aca:triggeredOnChange": false,
```